django_x509/base/admin.py

```diff
@@ -119,6 +120,21 @@ def crl_view(self, request, pk):
                             status=200,
                             content_type='application/x-pem-file')
 
+    def renew_ca(self, request, queryset):
+        renewed_rows = 0
+        for ca in queryset:
+            ca.renew_ca()
+            renewed_rows += 1
+        message = ngettext(
+            '%(renewed_rows)d CA has been successfully renewed',
```

Comment:
I think we could make more explicit the fact that also related certificates have been renewed, eg:

`CA and its related certificates have been successfully renewed`, maybe in this form we don't need to pluralize.

I also thought it would be good to have a confirmation step, but that can be done afterwards, so I created #85